src/pages/dev/amplifier/chain-integration/relay-messages/automatic.mdx

```diff
@@ -0,0 +1,33 @@
+# Automatically relay messages with the Amplifier API
+
+import { Callout } from "/src/components/callout"
+
+<Callout>
+If you wish to relay messages manually, check out [Manually relay messages through Amplifier](/dev/amplifier/chain-integration/relay-messages/).
+</Callout>
+
+Chains connected through the Amplifier must be able to send and receive GMP data with the Axelar Virtual Machine (AVM). The Amplifier API relayer is the component responsible for delivering this data. It does this through two distinct workflows:
+
+- Subscription (outgoing messages) — The relayer detects outgoing GMP messages from the chain to the AVM and publishes them to the GMP API so that they can be verified, routed, signed, and delivered to the destination chain.
+- Inclusion (incoming messages) — The relayer receives incoming GMP messages from the AVM to the chain and executes them by writing the transaction payloads to a block on the Amplifier chain.
+
+The integrated chain will have to provide a Subscriber and Includer to interact with the Amplifier API.
```

Comment:
again, I not sure for the wording we use in docs, so that might be a no-op, but mentioning it for context anyway:

In axelarons we have 3 components. Subscriber, Includer, and Listener. The Listener is the interaction point with the API. In summary
* Subscriber - detects events on a chain
* Includer - sends payloads to a chain
* Listener
  * receives events from the Subscriber and creates new tasks
  * polls for new tasks and constructs payloads for the Includer

Reply:
yeah, i was thinking the same thing. i updated the doc